### PR TITLE
#162163228: Registered users should be able to login.

### PR DESCRIPTION
**Pull request by KicaRonaldOkello**
#### What does this PR do?

    This pull enables a registered user to login to Authors Haven.
#### Description of Task to be completed?

- A registered user enters their email address and password, click the submit button, then they are signed into Authors Haven.

#### How should this be manually tested?

     `https://authors-heaven-react-pr-12.herokuapp.com/login`

#### What are the relevant pivotal tracker stories?
#162163228
#### Screenshots:
This is what is returned once a user submits an empty form
image

This is what is returned once a user submits either wrong email address or password.
image

### CODE REVIEW

File: src/views/LoginView/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      <LoginView 
+        onChange={this.handleChange}
+        onSubmit={this.handleSubmit}
+        errors={errors}
+      />
+    )
+  }
+}
+Login.propTypes = {
+  returnData: PropTypes.object,
```
[the remark this answers is not in the document]
Comment (JaymesKat) [reply src/views/LoginView/Login.jsx]: Sort this by using `Prototype.shape` [Example here](https://www.javascriptstuff.com/validate-nested-proptypes/)

**Review by joshtrigger — APPROVED** [review]

**Review by Nta1e — APPROVED** [review]
LGTM!

**Comment by Kasulejoseph** [comment]
typo `####What does this PR do?`

**Review by Kasulejoseph — APPROVED** [review]

**Review by JaymesKat — APPROVED** [review]
LGTM

**Pull request merged**